src/main/java/me/botsko/prism/Prism.java

```diff
@@ -394,6 +415,22 @@ public void onEnable() {
         });
     }
 
+    private Logger createPrismLogger(){
+        Logger result = Logger.getLogger("PrismLogger");
+        result.setUseParentHandlers(false);
+        for(Handler handler :result.getHandlers()){
```

Comment:
Codacy found an issue: [WhitespaceAround: 'for' is not followed by whitespace. Empty blocks may only be represented as {} when not part of a multi-block statement (4.1.3)](https://app.codacy.com/gh/AddstarMC/Prism-Bukkit/pullRequest?prid=5911179)